### ISSUE TITLE
[Usable – Make Code - Sign in with GitHub] Tooltip of ‘Help’ icon control is not visible via keyboard under ‘Sign in with GitHub' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3582]
**User Experience**:
User rely on keyboard will find difficulty to interact with the tooltip information if  tooltip is not accessible through keyboard as user won't be able to perform functionality over such control.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to "Your GitHub Repo" option and navigate through the all the controls of dialog "Sign in with GitHub"..
6. Verify the tooltip of visual icon control 'Help'’ by keyboard.

**Actual Result:**
Tooltip of ‘Help’ icon control is not visible via keyboard under ‘Sign in with GitHub' pop-up and due to this, end user is unable to get the context of that visual icons and also won't be able to perform functionality over such controls.

**Expected Result:**
Tooltip of ‘Help’ icon control should be displayed via keyboard under 'Sign in with GitHub' pop-up, so that User rely on keyboard will not find difficulty to interact with the tooltip information and perform functionality over such control.

**Note:**
1. This issue also exists with Firefox and Chrome Browser.
2. Same issue exists under 'Pair Device' pop-up.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Usable_Tooltip of help icon.zip](https://github.com/microsoft/pxt-microbit/files/5487379/Usable_Tooltip.of.help.icon.zip)

**Comment by jwunderl** [comment]
titles aren't displayed on focus in all browsers, but it looks like this is getting fixed in chromium currently: https://bugs.chromium.org/p/chromium/issues/detail?id=829352&can=2&q=os%3Dchrome
